tensorflow_addons/losses/README.md

```diff
@@ -1,4 +1,19 @@
 # Addons - Losses
 
+## Contents
+| Loss  | Reference                                              |
+|:----------------------- |:-------------------------------------|
+| TripletLoss | https://arxiv.org/abs/1503.03832 |
 
-## Standard API
\ No newline at end of file
+
+## Contribution Guidelines
+#### Standard API
+In order to conform with the current API standard, all losses
+must:
+ * Inherit from `keras.losses.LossFunctionWrapper`.
```

Comment:
Hmmm, no issue with adding that but want to clarify why it's necessary. My understanding of why we need to do this is so that Keras can properly serialize/deserialize objects. Are losses / optimizers saved as part of keras models? If so we should probably it to layers, losses, and optimizers

Reply:
Updated so we register all keras objects.